module/src/components/button/button.animated.scss

```diff
@@ -0,0 +1,7 @@
+.arm-button {
+  transition: opacity $transition-quick, background-color $transition-quick, transform $transition-quick;
+
+  &:active {
+    transform: translateY(2px);
```

Comment:
This replicates the bug in the old armstrong in safari where if you click the top of the button, the click fails to register as the mouseup event doesn't get called.

Reply:
haha good old safari

Reply:
I'll get rid of it and let it be up to the consuming app how the button animates